### PR TITLE
optional disable or product sync, refactor task calls

### PR DESCRIPTION
**Pull request by thomasmckay**
+ refactor task calls in content-views
+ publish and promote working of content-views
+ various locale strings
+ corrected export of act keys
+ --no-sync option and :products_no_sync: configuration

rubocop fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,6 +283,37 @@ Creating organization 'abcMega Corporation'... done
...
+* [Tests](https://github.com/Katello/hammer-cli-csv/blob/master/test/products_test.rb)
+* Sample data
+  * [Mega Corporation](https://github.com/Katello/hammer-cli-csv/blob/master/test/data/products.csv)
+* Supported products and version
+  * Katello-nightly, Katello-2.1, Katello-2.2
+  * Satellite-6.0, Satellite-6.1
+
+**CSV Columns**
+
+*Note: % column indicates Count substituion*
```
Comment (komidore64) [inline README.md]: can you explain this more?

File: lib/hammer_cli_csv/activation_keys.rb [hunk 2]
```diff
@@ -13,7 +13,9 @@ module HammerCLICsv
   class CsvCommand
     class ActivationKeysCommand < BaseCommand
       command_name 'activation-keys'
-      desc         'import or export activation keys'
+      desc         _('import or export activation keys')
+
+      option %w(--organization), 'ORGANIZATION', _('Only process organization matching this name')
 
       option %w(--organization), 'ORGANIZATION', 'Only process organization matching this name'
```
Comment (komidore64) [inline lib/hammer_cli_csv/activation_keys.rb]: did you mean to copy these?

File: lib/hammer_cli_csv/products.rb [hunk 3]
```diff
@@ -161,6 +159,20 @@ def content_type(repository_type)
       end
 
       def sync_repository(line, repository)
+        unless option_no_sync? || HammerCLI::Settings.get(:csv, :products_no_sync)
+          if option_verbose?
+            print _("sync'ing repository '%{repository_name}' in product '%{name}'...") %
+              {:repository_name => repository_name, :name => name}
+          end
+          if repository['sync_state'] == 'finished'
+            print _("previously sync'ed, skipping...") if option_verbose?
```
Comment (komidore64) [inline lib/hammer_cli_csv/products.rb]: past tense is `synced`

File: lib/hammer_cli_csv/products.rb [hunk 4]
```diff
@@ -161,6 +159,20 @@ def content_type(repository_type)
       end
 
       def sync_repository(line, repository)
+        unless option_no_sync? || HammerCLI::Settings.get(:csv, :products_no_sync)
+          if option_verbose?
+            print _("sync'ing repository '%{repository_name}' in product '%{name}'...") %
```
Comment (komidore64) [inline lib/hammer_cli_csv/products.rb]: `syncing`

File: README.md (hunk 1 above)
Comment (thomasmckay) [inline README.md]: added section to README.md

File: README.md [hunk 5]
```diff
@@ -283,6 +301,37 @@ Creating organization 'abcMega Corporation'... done
...
 
+## Products
+
+| Additional arguments | Description |
+| ---------------:| :--------------|
+| --organization | Only process organization matching this name |
+| --no-sync | Do not attempt to sync repositories |
+
+**Overview**
+* Due to the length of time that syncing repositories can take, the --no-sync option may be used to skip this step. To always disable syncing, ':products_no_sync: true' may be specified in configuration file.
```
Comment (komidore64) [inline README.md]: How about an option titled `sync_products` instead? By default, it's assumed to be `true`, and then the user sets it to `false` if they want to override the default. Having words like `no` or `disable` in options that are boolean has always seemed really confusing to me.

File: README.md (hunk 1 above)
Comment (komidore64) [inline README.md]: Ah, okay cool. thanks.

**Comment by komidore64** [comment]
ack

**Pull request merged**